daemon/src/projection.rs

```diff
@@ -691,8 +692,11 @@ pub struct CfdOrder {
     pub price: Price,
 
     /// Fee charged by the maker for opening a position
-    #[serde(with = "::bdk::bitcoin::util::amount::serde::as_btc")]
-    pub opening_fee: Amount,
+    ///
+    /// Note: It's the minimum possible fee, we cannot calculate the exact
```

Comment:
Hmm, this is unfortunate. I guess we need to do the same thing as we do for the margin. Calculate it per parcel and then calculate in the frontend.

Reply:
yes, exactly (still getting used to the new terminology!) - `quantity / `min_quantity` is parcel size after all :)